src/layouts/Layout.js

```diff
@@ -1,6 +1,7 @@
-import Header from "./Header";
-import Content from "./Content";
-import Footer from "./Footer";
+import Header from "../components/Header";
+import Content from "../components/Content";
+import Footer from "../components/Footer";
+
 import { BrowserRouter as Router } from "react-router-dom";
 
 const Layout = () => {
```

Comment:
You only need one <Router> on app core for basic routing, if it's in App you don't need it here

Reply:
Kept routing in App only.